tools/update_eval_results.py

```diff
@@ -1,574 +0,0 @@
-# Copyright (c) 2023 Intel Corporation
```

Comment:
We don't need this tool anymore?

Reply:
The script was outdate since https://github.com/openvinotoolkit/nncf/pull/1958#issuecomment-1641775114. 
Think that half a year is enough to make decision to remove it.

Reply:
@alexsu52 FYI

Reply:
+ @KodiaqQ

Reply:
We need to finalize the decision about the tool remove before removing it.
Even in the provided PR https://github.com/openvinotoolkit/nncf/pull/1958, there is a suggestion to track this in a separate ticket.
So, this PR shouldn't delete this.

Reply:
Ok, create 129482